deepcell_toolbox/utils.py

```diff
@@ -311,11 +311,14 @@ def resize(data, shape, data_format='channels_last', data_type='X'):
         else:
             shape = tuple(list(shape) + [data.shape[channel_axis]])
 
-        # linear interpolation (order 1) for image data, near neighbor (order 0) for labels
-        order = 1 if data_type == 'X' else 0
-
+        # linear interpolation (order 1) for image data, nearest neighbor (order 0) for labels
         # anti_aliasing introduces spurious labels, include only for image data
-        anti_aliasing = data_type == 'X'
+        if labeled_image:
+            order = 0
+            anti_aliasing = False
+        else:
+            order = 0
+            anti_aliasing = True
```

Comment:
```suggestion
        order = int(not labeled_image)
        anti_aliasing = int(not labeled_image)
```

Reply:
You could do it in one line too if they are always the same.

```python
order = anti_aliasing = int(not labeled_image)
```

Reply:
Yeah, good catch, I fixed the typo. Second one should be order=1

I agree that's more compact, but I worry it's getting a bit too fancy? I feel like the more verbose way takes up more space but is easier to understand

Reply:
Maybe instead:

```python
order = 0 if labeled_image else 1
anti_aliasing = not labeled_image
```

I see what you mean, but since we already have a boolean, might as well use it in the assignment logic. The following feels overly complicated:

```python
y = True
if y:
  x = True
else:
  x = False
```

Reply:
Oooh, yeah, I really like that version. I'll update it.